.gitignore

```diff
@@ -0,0 +1,5 @@
+.pio
+.idea
+CMake*
+CMakeListsPrivate.txt
+cmake-build-*/ 
```

Comment:
This file is unnecessary, we don't use cmake in this project. Also the others formats feel unimportant. Usually .gitignore files are used to put generated files by the project or build tools. And I don't see the case here for this project.